src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java

```diff
@@ -1,60 +1,91 @@
 package com.google.checkstyle.test.chapter4formatting.rule413emptyblocks;
 
-class InputEmptyBlocksAndCatchBlocks
-{
-    static {}
```

Comment:
If formatter changed this, then it seems like it is wrong.
https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.1.3-braces-empty-blocks
Also our google config wants a space between `{` and `}` which violate the same rule and the examples shown.

````
$ cat TestClass.java
/** Some javadoc. */
public class TestClass {
  static {} // violation

  static { } // no violation
}

$ java -jar checkstyle-10.17.0-all.jar -c /google_checks.xml TestClass.java
Starting audit...
[WARN] TestClass.java:3:10: WhitespaceAround: '{' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
[WARN] TestClass.java:3:11: WhitespaceAround: '}' is not preceded with whitespace. [WhitespaceAround]
Audit done.
````

Reply:
Moved to https://github.com/checkstyle/checkstyle/issues/15424 to investigate separately